ospd/ospd.py

```diff
@@ -1174,15 +1174,32 @@ def run(self) -> None:
         """ Starts the Daemon, handling commands until interrupted.
         """
 
+        self.scan_collection.data_manager = multiprocessing.Manager()
+
         try:
             while True:
                 time.sleep(SCHEDULER_CHECK_PERIOD)
                 self.scheduler()
                 self.clean_forgotten_scans()
+                self.check_pending_scans()
                 self.wait_for_children()
         except KeyboardInterrupt:
             logger.info("Received Ctrl-C shutting-down ...")
 
+    def check_pending_scans(self):
```

Comment:
If I read the method correctly the name should be changed because it doesn't check something for validity instead it created processes for pending scans. So better would be something like `start_pending_scans`